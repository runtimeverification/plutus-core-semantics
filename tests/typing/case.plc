(program 1.0.0 (abs a (type) (lam b (all a (type) (fun a (fun a a))) (lam t (fun (all a (type) (fun a a)) a) (lam f (fun (all a (type) (fun a a)) a)
  [ [ [ {b (fun (all b (type) (fun b b)) a)} t ] f ] (abs a (type) (lam x a x)) ] )))))