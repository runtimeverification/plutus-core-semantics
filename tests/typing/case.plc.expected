<generatedTop>
  <k>
    ( all a ( type ) ( fun ( all a ( type ) ( fun a ( fun a a ) ) ) ( fun ( fun ( all a ( type ) ( fun a a ) ) a ) ( fun ( fun ( all a ( type ) ( fun a a ) ) a ) a ) ) ) ) :: ( type )
  </k>
  <env>
    .
  </env>
</generatedTop>
